src/app/components/Login/login.tsx

```diff
@@ -1,19 +1,32 @@
-import { FormEvent, useState } from "react";
+import { FormEvent, useEffect, useState } from "react";
 import styles from "./login.module.css";
 
 export default function Login(): JSX.Element {
-  const [name, setName] = useState("");
+  const [name, setName] = useState(
+    sessionStorage.getItem("Current User") || ""
+  );
+
+  useEffect(() => {
+    sessionStorage.setItem("Current User", name);
+  }, [name]);
 
   const handleSubmit = async (event: FormEvent) => {
     event.preventDefault();
-
-    await fetch("/api/users", {
-      method: "POST",
-      headers: {
-        "Content-Type": "application/json",
-      },
-      body: JSON.stringify({ name: name }),
-    });
+    const response = await fetch(`/api/users/${name}`);
+    console.log(response);
+    const isUserKnown = await response.text();
+    console.log(isUserKnown);
+    if (isUserKnown) {
```

Comment:
If you apply the error code in your backend, you can then check here if the response is ok.
For a detailed description, take a look here: https://javascript.info/fetch

```suggestion
    if (response.ok) {
```